packages/wallet-sdk/src/CoinbaseWalletProvider.ts

```diff
@@ -53,6 +54,16 @@ export class CoinbaseWalletProvider extends ProviderEventEmitter implements Prov
             storeSignerType(signerType);
             break;
           }
+          case 'wallet_sendCalls':
```

Comment:
What happens if an unconnected EOA calls these methods. i.e. I have the extension

Reply:
tested locally, these rpc methods are currently not supported for EOA and in extension. the request will be passed over to extension, but extension will reject with unsupported error at serviceWorker lever
![image](https://github.com/user-attachments/assets/4e59467e-b5ee-49d9-945c-967c64a76f50)

Reply:
Great!